src/frontend/pages/game.tsx

```diff
@@ -0,0 +1,171 @@
+import Head from 'next/head';
+import styles from '../styles/Game.module.css';
+import Slider from 'react-input-slider';
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+
+export default function Home() {
+  const [takeVal, setTakeVal] = useState(10);
+  const [timeLeft, setTimeLeft] = useState(10);
```

Comment:
could we use a starter constant here, I feel like we're probably going to end up referencing the ten second interval and ten take limit in other places? Plus it will be better so looking back we'll know what each 10 means